py3status/modules/weather_owm.py

```diff
@@ -0,0 +1,741 @@
+# -*- coding: utf-8 -*-
+'''
+Ultimately customizable weather module based on the IP-API Geolocation API
+(http://ip-api.com) and the OpenWeatherMap API (https://openweathermap.org).
+
+Requires an API key for OpenWeatherMap (OWM), but the free tier allows you
+enough requests/sec to get accurate weather even up to the second.
+
+This module allows you to specify an icon for nearly every weather scenario
+imaginable. The default configuration options lump many of the icons into
+a few groups, and due to the limitations of UTF-8, this is really as expressive
+as it gets.
+
+I would highly suggest you install an additional font, such as the incredible
+(and free!) Weather Icons font (https://erikflowers.github.io/weather-icons),
+which has icons for most weather scenarios. But, this will still work with
+the i3bar default font, Deja Vu Sans Mono font, which has Unicode support.
+You can see the (limited) weather icon support within Unicode in the defaults.
+
+For more information, see the documentation
+(https://openweathermap.org/weather-conditions) on what weather conditions are
+supported. See the configuration options for how to specify each weather icon.
+
+Configuration parameters:
+    api_key: Your OpenWeatherMap API key
+        See https://openweathermap.org/appid. Required!
+        (default None)
+    cache_timeout: The time between weather polling in seconds
+        It is recommended to keep this at a higher value to avoid rate
+        limiting with the API's.
+        (default 600)
+    forecast_days: Number of days to include in the forecast, including today
+        (regardless of the 'forecast_include_today' flag)
+        (default 0)
+    forecast_include_today: Include today in the forecast? (Boolean)
+        (default False)
+    forecast_text_separator: Separator between entries in the forecast
```

Comment:
This was left as `forecast_text_separator` to have better cross compatibility with `weather_yahoo`.